hack/bundle-gen.py

```diff
@@ -325,14 +389,17 @@ def open_pr(
 
     print()
     print()
-    print("Cloning %s" % fork_repo)
     repo_full_path = os.path.join(work_dir, dir_name)
-    # clone git repo
-    try:
-        git.Repo.clone_from(fork_repo, repo_full_path)
-    except:
-        print("Failed to clone repo {} to {}".format(fork_repo, repo_full_path))
-        raise
+    if not os.path.exists(repo_full_path):
```

Comment:
Ah, nice solution. My only quibble is that the connection between `dir_name` here and [above](https://github.com/openshift/hive/pull/2056/files#diff-625239517ac654a0a60ffdcc20e3a62f806a20df37b553a2ce9e700b57db8176R136) is a bit brittle. I'll say good enough for now, but worth adding a comment in one or both places referring to the other and explaining the connection.